src/main.rs

```diff
@@ -66,6 +66,12 @@ async fn handle_message(
     // Accounts
     db_adapters::accounts::handle_accounts(&pool, &streamer_message.receipt_execution_outcomes)
         .await;
+    // AccessKeys
+    db_adapters::access_keys::handle_access_keys(
+        &pool,
+        &streamer_message.receipt_execution_outcomes,
+    )
+    .await;
```

Comment:
I guess we can already parallelize some of the handlers (execution outcomes, accounts, access keys are independent)